packages/create-next-app/README.md

```diff
@@ -1,8 +1,32 @@
-# create-next-app
+# Create Next App
 
-This package includes the global command for creating [Next.js](https://github.com/vercel/next.js) applications.
+The easiest way to get started with Next.js is by using `create-next-app`. This simple CLI tool enables you to quickly start building a new Next.js application, with everything set up for you. You can quickly create a new app using the default Next.js template, or choose from one of the <a aria-label="Next.js Starter Templates" href="https://github.com/vercel/next.js/tree/canary/examples">templates available here.</a> 
 
-Please refer to its documentation:
+The following command will start the interactive experience for using `create-next-app`. The interactive experience will walk you through the process of creating the new app, and you can choose from the available templates directly from the command line:
```

Comment:
This is quite redundant with the text that's above 🤔

Reply:
You are right!